src/main/java/com/yanna/stepanova/service/impl/UserServiceImpl.java

```diff
@@ -35,7 +43,19 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto)
         User user = userMapper.toModel(requestDto);
         user.setPassword(passwordEncoder.encode(user.getPassword()));
         user.setRoles(generateDefaultSetRoles());
-        return userMapper.toResponseDto(userRepo.save(user));
+        User savedUser = userRepo.save(user);
```

Comment:
```suggestion
        shoppingCartService.createShoppingCart(user);
```
Create method in ShoppingCartServiceImpl
```
@Override
    public void createShoppingCart(User user) {
        ShoppingCart shoppingCart = new ShoppingCart();
        shoppingCart.setUser(user);
        repository.save(shoppingCart);
    }
```